config/sample_settings/general.json

```diff
@@ -3,7 +3,9 @@
         "name": "Tooling Account [<<env>>]",
         "account_id": "<<tooling_account_id>>",
         "region": "eu-central-1",
-        "metrics_collection_interval_min" : 5
+        "metrics_collection_interval_min" : 5,
+        "grafana_vpc_id": "<<vpc_id>>", 
```

Comment:
For me, it looks a bit more logical to have a separate section for Grafana:
"grafana_instance" : {
   "vpc_id" :
   "security_group_id" :
}
If we "grafana_instance" is present is general, then we create a stack. And we don't otherwise.

Reply:
yes, good idea :)

Reply:
A separate section added